src/main/drivers/bus_spi.c

```diff
@@ -151,27 +163,46 @@ bool spiBusWriteRegister(const busDevice_t *bus, uint8_t reg, uint8_t data)
     return true;
 }
 
-bool spiBusReadRegisterBuffer(const busDevice_t *bus, uint8_t reg, uint8_t *data, uint8_t length)
+bool spiBusRawReadRegisterBuffer(const busDevice_t *bus, uint8_t reg, uint8_t *data, uint8_t length)
+
 {
     IOLo(bus->busdev_u.spi.csnPin);
-    spiTransferByte(bus->busdev_u.spi.instance, reg | 0x80); // read transaction
+    spiTransferByte(bus->busdev_u.spi.instance, reg);
```

Comment:
Wait what? Is this the change that was needed to make the A7105 driver work?

Reply:
Yeap.

A7105 seems to use 0x40 bit, unlike lots of others that use 0x80, to indicate a read cycle.

Reply:
CC2500 uses 0x80
https://github.com/betaflight/betaflight/blob/f645ba3c10e4ebb4da831300a707d020d33e2daa/src/main/drivers/rx/rx_cc2500.c#L72-L75

A7105 uses 0x40
https://github.com/betaflight/betaflight/blob/f645ba3c10e4ebb4da831300a707d020d33e2daa/src/main/drivers/rx/rx_a7105.c#L125-L128

nRF24L01 is completely different
https://github.com/betaflight/betaflight/blob/f645ba3c10e4ebb4da831300a707d020d33e2daa/src/main/drivers/rx/rx_nrf24l01.c#L50-L52
https://github.com/betaflight/betaflight/blob/f645ba3c10e4ebb4da831300a707d020d33e2daa/src/main/drivers/rx/rx_nrf24l01.c#L97-L100

These called for the `Raw` version.